tests/common/test_event.py

```diff
@@ -401,3 +401,36 @@ def test_report_metric(self, mock_event):
                 break
         else:
             self.fail("Counter '%idle' not found in event parameters: {0}".format(repr(event_dictionary)))
+
+    def test_cleanup_message(self):
```

Comment:
Can you add a test with a log message starting with `[PERIODIC]`?

Reply:
Good suggestion. Added in the tests.